lib/seadsa/DsaGlobal.cc

```diff
@@ -104,8 +105,7 @@ bool ContextInsensitiveGlobalAnalysis::runOnModule(Module &M) {
     // --- all scc members share the same local graph
     for (CallGraphNode *cgn : scc) {
       Function *fn = cgn->getFunction();
-      if (!fn || fn->isDeclaration() || fn->empty())
-        continue;
+      if (!fn || fn->isDeclaration() || fn->empty()) continue;
```

Comment:
I think we only pass `specGraphInfo` to two analyses:  BottomUpTopDown and ContextSensitive. We should also pass it to all derived classes from `GlobalAnalysis` and `DsaGlobalPass` that includes the context insensitive variant and the flat one.

Reply:
Also `CompleteCallGraphAnalysis` would benefit from `specGraphInfo`

Reply:
53ec4cc